api/src/ContinuousPipe/Builder/GoogleContainerBuilder/HttpBuildCreator.php

```diff
@@ -29,16 +29,23 @@ class HttpBuildCreator implements BuildCreator
      */
     private $maximumAllowedBuildTime;
 
+    /**
+     * @var string
+     */
+    private $builderImage;
+
     public function __construct(
         GuzzleHttpClientFactory $httpClientFactory,
         string $googleProjectId,
         string $googleSourceArtifactBucket,
-        int $maximumAllowedBuildTime = 3600
+        int $maximumAllowedBuildTime = 3600,
+        string $builderImage = 'gcr.io/continuous-pipe-1042/cloud-builder'
```

Comment:
Same, let's use the Docker Hub one here.

Reply:
Ok

What do you think about putting allowed build time to the and and break BC?

Reply:
> What do you think about putting allowed build time to the and and break BC?

I don't get the question :)

Reply:
Well, as long as you change the service definition, that should be OK. I don't know anyone having such a deep customisation yet so I believe we can go ahead.